src/font-loader.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Polyfill loosely based on zenfonts
+ * https://github.com/zengabor/zenfonts/
+ * Copyright (c) 2015 Gabor Lenard
+ */
+
+/* global document */
+
+const testFonts = ['sans-serif', 'serif', 'monospace'];
+const testText = 'π+[.—_]imW12/?'; // FIXME: allow override
+// how long to wait for load before switching style to true font anyway
+const timeout = 60000;
+
+export default function(font) {
+  if (__BROWSER__) {
+    return !document.fonts
```

Comment:
Is this ternary correct? Seems like it should be the other way around

Reply:
Good catch. I had it backwards to test the polyfill in chrome